app/src/main/java/net/gsantner/markor/util/DocumentIO.java

```diff
@@ -45,7 +45,7 @@
     public static final String EXTRA_PATH_IS_FOLDER = "EXTRA_PATH_IS_FOLDER"; // boolean
 
     public static final int MAX_TITLE_EXTRACTION_LENGTH = 25;
-    public static boolean SAVE_IGNORE_EMTPY_NEXT_TIME = false;
+    public static boolean SAVE_IGNORE_EMTPY_NEXT_TIME = true;
```

Comment:
@gsantner Why do we prevent saving empty files? I can revert this too but I made the change so I could ask.

Reply:
Please for sure revert.

For reason, some devices have either slow disk i/o OR slow cpu filling edittext slow. In either combination it may (and has happened) to result in empty files being written.